libs/styles/styles.css

```diff
@@ -892,6 +892,14 @@ a.static:active  {
   visibility: hidden;
 }
 
+/* Anchors */
+span.anchor:target::before {
+  content: "";
+  display: block;
+  height: 100px;
+  margin: -100px 0 0;
+}
```

Comment:
This should be revised. Currently, it makes the content around the anchor point unselectable, which is not ideal. "After Effects" and the eyebrow text can't be selected in the screenshot below.
<img width="843" alt="Screenshot 2025-02-18 at 13 56 37" src="https://github.com/user-attachments/assets/8fd145e2-87fa-49e5-8085-4505c40fdd49" />

Reply:
A simple negative `z-index` value (and relative position) or a `pointer-events: none;` should do the trick.

Reply:
Great finding, thank you! I will also fix the existing code for marquee-anchors.